### PR TITLE
Pengu's winter coats. V3.

### PR DESCRIPTION
**Pull request by Homingpenguins**
## About The Pull Request
The third time ive done this. Adds 4 winter coats.
Chem, Gene, Barkeep, and Lawyer.
Thanks to Jo, YouTubeBoy, and BeardlessJo for making the Lawyer, Barkeep, and Genetic coats respectively.

## Why It's Good For The Game
It get us further to the goal of having winter coats for all jobs.

## Changelog
:cl:
add: files for organization.
imageadd: added some icons and images : the 5 dmis
icons>Fulpicons>Penguins_cove>Winter_Coat_Pile> (the 5 dmis)
code: changed some code : 
code>vendor>wardrobe.dm (A /tg/ file)
code>Fulpcode>wintercoats.dm (code for winter coats
/:cl:

### CODE REVIEW

File: code/FulpstationCode/wintercoats.dm [hunk 1]
```diff
@@ -104,3 +104,79 @@
 	name = "\improper Quarter Master's hardsuit"
 	desc = "Might as well make your own hardsuit if CentCom won't give you one."
 */
+
+//this starts Homingpenguins' section of winter coat code.
+//Sprites made by Papaporo Paprito (again), Jo, YouTubeBoy, HomingPenguins, and BeardlessJo
+//code done by Homingpenguins.
+
+/obj/item/clothing/suit/hooded/wintercoat/security/lawyer
+	name = "\improper Lawyer's winter coat"
```
Comment (Shadowtail117) [inline code/FulpstationCode/wintercoats.dm]:
```suggestion
	name = "\improper lawyer's winter coat"
```
All other items are lowercase. Let's maintain consistency.

File: code/FulpstationCode/wintercoats.dm [hunk 2]
```diff
@@ -104,3 +104,79 @@
...
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/security/law
+
+/obj/item/clothing/head/hooded/winterhood/security/law
+	mob_overlay_icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoats.dmi'
+	icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoathoods.dmi'
+	icon_state = "winterhood_law"
+
+/obj/item/clothing/suit/hooded/wintercoat/service/bartender
+	name = "\improper Bartender's winter coat"
```
Comment (Shadowtail117) [inline code/FulpstationCode/wintercoats.dm]:
```suggestion
	name = "\improper bartender's winter coat"
```
See above.

File: code/FulpstationCode/wintercoats.dm [hunk 3]
```diff
@@ -104,3 +104,79 @@
...
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/service/bar
+
+/obj/item/clothing/head/hooded/winterhood/service/bar
+	mob_overlay_icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoats.dmi'
+	icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoathoods.dmi'
+	icon_state = "winterhood_bar"
+
+/obj/item/clothing/suit/hooded/wintercoat/medical/chem
+	name = "\improper Chemist's winter coat"
```
Comment (Shadowtail117) [inline code/FulpstationCode/wintercoats.dm]:
```suggestion
	name = "\improper chemist's winter coat"
```
See above.

File: code/FulpstationCode/wintercoats.dm [hunk 4]
```diff
@@ -104,3 +104,79 @@
...
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 50, "rad" = 0, "fire" = 50, "acid" = 50)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/medical/chem
+
+/obj/item/clothing/head/hooded/winterhood/medical/chem
+	mob_overlay_icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoats.dmi'
+	icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoathoods.dmi'
+	icon_state = "winterhood_chem"
+
+/obj/item/clothing/suit/hooded/wintercoat/medical/gen
+	name = "\improper Geneticist's winter coat"
```
Comment (Shadowtail117) [inline code/FulpstationCode/wintercoats.dm]:
```suggestion
	name = "\improper geneticist's winter coat"
```
See above.

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]
All items in the game are lowercase. These winter coats being capitalized is a glaring contradiction.

Other than that, these look fine.

File: code/FulpstationCode/wintercoats.dm (hunk 4 above)
Comment (Homingpenguins) [reply code/FulpstationCode/wintercoats.dm]: fixed

File: code/FulpstationCode/wintercoats.dm (hunk 3 above)
Comment (Homingpenguins) [reply code/FulpstationCode/wintercoats.dm]: fixed

File: code/FulpstationCode/wintercoats.dm (hunk 2 above)
Comment (Homingpenguins) [reply code/FulpstationCode/wintercoats.dm]: fixed

File: code/FulpstationCode/wintercoats.dm (hunk 1 above)
Comment (Homingpenguins) [reply code/FulpstationCode/wintercoats.dm]: fixed

**Review by Shadowtail117 — APPROVED** [review]

**Comment by Homingpenguins** [comment]
thanks fixed those problems

**Comment by TheSwain** [comment]
@Homingpenguins okay to merge this, one of the standards we use when adding objects to existing TG lists is to put them somewhere above the end of the list. basically, any time an item appears in a vending machine or anything like that, it can't be the last thing in the list or we'll get merge conflicts. if you can move them up a few entries any time they're added to the world, that'll make us all set

**Pull request closed without merging**